packages/react/src/Link/Link.module.css

```
@@ -18,10 +17,9 @@
   bottom: -0.15em;
   left: 0;
   width: calc(100% - var(--arrow-width));
-  height: 2px;
   pointer-events: none;
   content: '';
-  background-color: currentColor;
+  border: 1px solid currentColor;
```

We have a css var for this:

```suggestion
  border: var(--brand-borderWidth-thin) solid currentColor;
```

---

"There is an app for that" pun huh? 😆